src/lfo.cpp

```diff
@@ -100,17 +100,17 @@ ParamsAfterLfo lfoTick() {
 		lfoDepthLfo6 = 0;
 	}
 
-	lfoDepth[0] = preset_data.lfo[0].depth + lfoDepthLfo1 + lfoDepthLfo2;
```

Comment:
The comment suggests that the mod wheel value is being added here. Is it a bug that this term is missing here or does the modwheel affect `preset_data.lfo[0].depth` directly?

Reply:
modwheel does indeed want to modulate the lfo1 depth here, works as expected unless I missed something

Reply:
ahh I see how it's done now. The way you are currently doing it implies that the modwheel *changes* / *overrides* the `lfo[0].depth` value (i.e. alters the preset which would be noticeable when saving).

Why don't you do it like with `velocityToLfo` and `aftertouch`, i.e. storing the current modwheel's value in a variable, and incorporating that variable in the `lfoDepth[0]` calculation here?

This would have the advantage that the modwheel does not alter the preset, and that the modwheel actually *adds* to `lfo[0].depth`, rather than overwriting it.

Reply:
makes sense